packages/core/src/data-module/data-cache/dataReducer.ts

```diff
@@ -171,8 +176,9 @@ export const dataReducer: Reducer<DataStreamsStore, AsyncActions> = (
           : requestCache,
         dataCache: updatedDataCache,
         isLoading: false,
-        isRefreshing: !!restOfDataStream.isRefreshing,
         error: undefined,
+        numOutgoingRequests: noOfOutGoingrequest,
+        isRefreshing: noOfOutGoingrequest ? noOfOutGoingrequest > 0 : false,
```

Comment:
can we use like this " isRefreshing: noOfOutGoingrequest > 0,"  ?

Reply:
resolved